Ghidra/Processors/68000/data/languages/68000.pspec

```diff
@@ -2,7 +2,6 @@
 
 <processor_spec>
   <properties>
-    <property key="emulateInstructionStateModifierClass" value="ghidra.program.emulation.m68kEmulateInstructionStateModifier"/>
```

Comment:
I shouldn't be removing this, as `m68kEmulateInstructionStateModifier` now contains `findFirstOne`. Before it only had one for countLeadingZeros, so I deleted it.